web/sso.cert

```diff
@@ -0,0 +1 @@
+-----BEGIN CERTIFICATE-----
MIIDpDCCAoygAwIBAgIGAXE5JNWbMA0GCSqGSIb3DQEBCwUAMIGSMQswCQYDVQQGEwJVUzETMBEG
A1UECAwKQ2FsaWZvcm5pYTEWMBQGA1UEBwwNU2FuIEZyYW5jaXNjbzENMAsGA1UECgwET2t0YTEU
MBIGA1UECwwLU1NPUHJvdmlkZXIxEzARBgNVBAMMCmRldi0yMTUzNDQxHDAaBgkqhkiG9w0BCQEW
DWluZm9Ab2t0YS5jb20wHhcNMjAwNDAyMDQyNjI3WhcNMzAwNDAyMDQyNzI3WjCBkjELMAkGA1UE
BhMCVVMxEzARBgNVBAgMCkNhbGlmb3JuaWExFjAUBgNVBAcMDVNhbiBGcmFuY2lzY28xDTALBgNV
BAoMBE9rdGExFDASBgNVBAsMC1NTT1Byb3ZpZGVyMRMwEQYDVQQDDApkZXYtMjE1MzQ0MRwwGgYJ
KoZIhvcNAQkBFg1pbmZvQG9rdGEuY29tMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA
tE5rwNDKx6m95Qp1C/s+ICcCbjUMMCWm6w1lfRw4MA0Sky4eg4yeQnCMq6yBvy6HG8bykpDSRlHo
1bDrVfQJSrk4tg4KvN4tpZIxijyXhLFLztXV64BIOTgFyifgOYZnDRUmp9pGiTAnkZMn9iUUpz/a
WfA0MUiLs7RHmwdGdbsMToxYx1MDukQUx2LPz3AYFLnPoqu6FYQVNngtuwFRe7itbCX0DSycfrBP
KeUrmpmNYGoRDVZgW0fcIlUGhMRxe/J6YW9GX0X15rjClWPPuraaVkCF7eH9MGnoJ3+oCpENMoml
xF/FEic+2xhMp9qeR3sXxT9cto9DL6fwQry6/wIDAQABMA0GCSqGSIb3DQEBCwUAA4IBAQCeEeHP
VdGKbDWqlRysVl+eCBoznin9FR95lObfpVxJinLqMwSaKnir8/2uTbWxiOwPaPxw4M8RmZ3Pt+Cs
0H5IbFFG+nN1vsSJU4iCjomxb/1cTtzhU0hcJCYczITFDO1T0kBr3Tp2Y+b9BQj7FJEcrJXMNcjr
Z7CIesTUkaNRtLDSuO+E60mCWE7ahbW4fBkjbip9aymnSPzoCunvO0VeIvDqqrvTlhG/DOfH/ytZ
LOEjb9wZ7zOXFlgd7PeFNwmbc8B9NtXgOWlKDEWTzrN3jogD1kP0blfOhFcUGDIdTw2+bP7tzplw
MQObFk7lsnSblCOw9yKfDftgyhF2noEG
-----END CERTIFICATE-----

```

Comment:
As we discussed this certificate should not be included here.

But there needs to be a instructions to generate the certificates and if someone would like to use the functionality, they can follow the instructions to generate the certificate, add them to the file and run the docker-compose.